packages/illustrations/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import products from './products'
+
+export default {
+  products,
```

Comment:
I don't know if you saw but the ticket got new comment in the structure, we need to have `product.instance`, `product.database` etc